mobile/src/bundles/common/components/selector/selector.tsx

```diff
@@ -70,7 +94,9 @@ const Selector = <T extends FieldValues>({
                 onPress={toggleIsListVisible}
             >
                 <Text category={TextCategory.LABEL}>
-                    {selectedOption ?? placeholder}
+                    {selectedOptions.length > NO_SELECTED
+                        ? selectedOptions.join(', ')
+                        : placeholder}
                 </Text>
```

Comment:
Have to write like this because of error: 
Unnecessary conditional, expected left-hand side of `??` operator to be possibly null or undefined  @typescript-eslint/no-unnecessary-condition  if write like this `selectedOptions.join(',  ) ?? placeholder`